educe/rst_dt/learning/cmd/extract.py

```diff
@@ -9,39 +9,25 @@
 """
 
 from __future__ import print_function
-import codecs
-import csv
 import os
 import sys
 
 import educe.corpus
-import educe.learning.keys
 import educe.glozz
 import educe.stac
 import educe.util
 from ..args import add_usual_input_args
 from ..base import read_corpus_inputs, extract_pair_features
-
-NAME = 'extract'
+from ....learning.orange_format import dump_orange_tab_file
```

Comment:
I'm not entirely convinced it makes sense to use a relative import here.  This is one area of code I'm horribly inconsistent about because I haven't yet figured out what a good policy is for the use of one or the other.  Part of the attraction to relative imports is that it can make code a bit more robust wrt evolution like modules being moved around… provided of course the imports live in the same subtree or below.

But taken beyond the `.` and `..` level, I think we maybe run into the problem of relative imports being rather obscure.